README.md

```diff
@@ -65,15 +65,25 @@ winget install --id Kitware.CMake
 winget install --id Microsoft.VisualStudio.2022.BuildTools --force --override "--passive --wait --add Microsoft.VisualStudio.Workload.VCTools;installRecommended --add Microsoft.VisualStudio.Component.VC.Llvm.Clang --add Microsoft.VisualStudio.Component.VC.Llvm.ClangToolset"
 ```
 
-### Step 1: Obtain model weights and tokenizer from Kaggle
+### Step 1: Obtain model weights and tokenizer from Kaggle or Hugging Face Hub
 
 Visit [the Gemma model page on
-Kaggle](https://www.kaggle.com/models/google/gemma) and select `Model Variations
+Kaggle](https://www.kaggle.com/models/google/gemma/frameworks/gemmaCpp) and select `Model Variations
 |> Gemma C++`. On this tab, the `Variation` dropdown includes the options below.
 Note bfloat16 weights are higher fidelity, while 8-bit switched floating point
 weights enable faster inference. In general, we recommend starting with the
 `-sfp` checkpoints.
 
+Alternatively, visit the [gemma.cpp](https://huggingface.co/models?other=gemma.cpp)
+models on the Hugging Face Hub. First go the the model repository of the model of interest
+(see recommendations below). Then, click the `Files and versions` tab and download the 
+model and tokenizer files. For programmatic downloading, if you have `huggingface_hub`
+installed, you can also run:
+
+```
+huggingface-cli download google/gemma-2b-cpp --local-dir build/
```

Comment:
Few minor tweaks:

- Use `google/gemma-2b-sfp-cpp` as the example here instead of `google/gemma-2b-cpp`
- Add a comment about `login`, I ran into the authentication error the first time I ran this command.
- Can you add a `models/` directory with a `.gitignore` placeholder and use that as the example local directory? Want to nudge people towards that going forwrd since `build/` gets wiped when changing configurations.

Reply:
Done the first two points. I'm downloading the model to `build/` as that's where the models are moved in Kaggle. Should we update all the instructions from `build` to `models` in a follow-up PR?

Reply:
That would be helpful thanks, yes kaggle and HF instructions should probably both point to a directory that's not `build/`.

Reply:
Should we merge this PR ahead and then I open a follow-up PR? I can also open a PR against this PR, but I think the first would be cleaner